examples/jsm/nodes/materials/NodeMaterial.js

```diff
@@ -44,6 +44,9 @@ class NodeMaterial extends ShaderMaterial {
 
 		this.positionNode = null;
 
+		this.vertexShader = null;
```

Comment:
I think these properties are already created by ShaderMaterial?

Reply:
They have the text for a minimal shader applied.
https://github.com/mrdoob/three.js/blob/8d2430fb0a995742844ed667710de43a73d74c54/src/materials/ShaderMaterial.js#L21-L22

setting these to null ensures the new 'shader missing' code in getParameters() is triggered and hence material.onBuild()